projects/js-packages/charts/src/components/conversion-funnel-chart/test/conversion-funnel-chart.test.tsx

```diff
@@ -47,7 +48,7 @@ describe( 'ConversionFunnelChart', () => {
 			renderWithoutTheme( <ConversionFunnelChart { ...defaultProps } /> );
 
 			// Check main rate is displayed (first occurrence)
-			expect( screen.getAllByText( '10.3%' ) ).toHaveLength( 2 ); // Main rate + Purchase step
+			expect( screen.getAllByText( formatPercentage( 10.3 ) ) ).toHaveLength( 2 ); // Main rate + Purchase step
```

Comment:
Are these changes here intentional?

Reply:
No worries, I can easily fix this back to literal 👍 😄